.gitignore

```diff
@@ -9,4 +9,5 @@ bundles
 *.DS_Store
 .eggs
 dist
-**/*.egg-info
\ No newline at end of file
+**/*.egg-info
+.vscode
```

Comment:
Please remove, you can keep this in a local `.gitignore`.

Reply:
googling "local gitignore" led me to understand how to use the .git/info/exclude, so thank you for that! I now know how to avoid this mistake in the future.